test/read_config_test.go

```diff
@@ -60,3 +64,29 @@ func TestRead_ReadAndWriteTimeoutConfig(t *testing.T) {
 		t.Fail()
 	}
 }
+
+func TestRead_EnableFunctionReadinessProbeConfig(t *testing.T) {
```

Comment:
One more test please for the "default" which should be on.

Reply:
@alexellis fixed.

Reply:
Perfect 👍 really pleased with this work, good job